src/style.css

```
@@ -1,3 +1,87 @@
+:root {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  --bg-color: #fff;
+  --a-color: #000;
+  --a-hover-color: #00a0e3;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url("Roboto/Roboto-Medium.ttf");
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  color: var(--a-color);
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--bg-color);
+  padding-top: 10px;
+  padding-left: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 22.5px;
+  justify-content: flex-end;
+  list-style: none;
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  margin-top: 12.5px;
+  padding-top: 0;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  padding: 0 20px;
```

You need padding top & bottom, not left and right.

More like this:
```suggestion
  padding: 20px 0;
```